src/golang/cmd/migrator/migrator/register.go

```diff
@@ -11,6 +11,7 @@ import (
 	_000006 "github.com/aqueducthq/aqueduct/cmd/migrator/versions/000006_add_retention_policy_column"
 	_000007 "github.com/aqueducthq/aqueduct/cmd/migrator/versions/000007_workflow_dag_edge_pk"
 	_000008 "github.com/aqueducthq/aqueduct/cmd/migrator/versions/000008_delete_s3_config"
+	_000009 "github.com/aqueducthq/aqueduct/cmd/migrator/versions/000009_metadata_interface_backfill"
```

Comment:
Can you pull the migrator changes to a follow up PR? This PR is already quite big - I can review the rest but it makes more sense to have someone like saurav look at the migrator-specific stuff more closely.

Reply:
You'll also generally get a faster turnaround time with smaller PRs :)